src/shell/shell.zig

```diff
@@ -2549,10 +2721,11 @@ pub fn handleTemplateValue(
             const last = array.len -| 1;
             var i: u32 = 0;
             while (array.next()) |arr| : (i += 1) {
-                if (!(try handleTemplateValue(globalThis, arr, out_jsobjs, out_script, jsobjref_buf))) return false;
+                if (!(try handleTemplateValue(globalThis, arr, out_jsobjs, out_script, jsstrings, jsobjref_buf))) return false;
                 if (i < last) {
                     const str = bun.String.init(" ");
-                    if (!try appendBunStr(str, out_script, false)) return false;
+                    defer str.deref();
```

Comment:
you can use `String.static` here. it is identical to init with a string literal.

the defer deref is smart but in this case it doesn't really do anything interesting.